CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ This release closes the [8.0.0 milestone](https://github.com/jessesquires/JSQMes
 - Dynamic text support. (#497, #1747) Thanks @MacMeDan!
 - Message cells now have a customizable accessory view. (#1519, #1719) Thanks @adubr!
 - Send button now can be turned on/off manually. (#1575 #1609) Thanks @sebastianludwig!
+- Video bubble now can have a custom thumbnail. (#628 #709 #1408) Thanks @weekwood @benjaminhallock
```

Comment:
Ah, yes: 😊 

Let's have 1 method that accepts a param for the frame, and a convenience method that provides a default. 

Let's spell them like this:

- `thumbnailFromVideoMediaItem: completion:`
- `thumbnailFromVideoMediaItem: time: completion:`

(The first method should just call the second with a default `time` value)